### DISCUSSION TITLE
Does filament support alpha hash

### DISCUSSION THREAD

**Discussion by dandingol03** [google/filament#7252] [Q&A]
Hi. I found `blender` support alpha hash in transparent material,  but `filament` use alpha blend to render transparent stuff. Does filament support alpha hash? If not, how can i implement alpha hash in `filament`

**Comment by romainguy** [comment]
You can probably do it yourself using an alpha masked material.

**Reply by dandingol03** [reply]
@romainguy  Does the `alpha masked material` mean the mat which is setted as below?

 ` 
material {
    blending : masked,
    maskThreshold : value
}
 ` 
and the maskThreshold is generated by alpha hash. Am i right?

**Reply by dandingol03** [reply]
i can't set a fixed maskThreshold in the mat file, because `alpha hash` depend on random alpha threshold. This alpha threshold is supposed to be placed in depth_main.fs, and discard fragment if the fragment's alpha value lower than alpha hash threshold.
